docs/blog/2022-04-29-faststore.md

```diff
@@ -0,0 +1,18 @@
+---
+description: New Next.js starter available. This starter's features are identical to the existing base store starter, which used Gatsby.
+tags: [faststore]
+---
+
+# New Next.js starter available
+
+VTEX launched a new [Next.js based FasStore starter](https://github.com/vtex-sites/nextjs.store). It is actually a base store starter, identical in every way to the existing base store, which used Gatsby, except for the framework.
+
+## What has changed?
+
+Now, developers that wish to use Next.js with FastStore, can leverage our base store starter to get their projects going.
+
+Also, because this is another base store starter, [the name of the repository formerly known as `basestore` was changed to `gatsby.store`](https://faststore.dev/releases/2022/04/29/basestore).
```

Comment:
```suggestion
Also, because the Base Store is now available for Next.js and Gatsby, [the name of the repository formerly known as `base.store` was changed to `gatsby.store`](https://faststore.dev/releases/2022/04/29/basestore).
```